tool.go

```diff
@@ -264,6 +264,7 @@ func main() {
 	run := cmdTest.Flags().String("run", "", "Run only those tests and examples matching the regular expression.")
 	short := cmdTest.Flags().Bool("short", false, "Tell long-running tests to shorten their run time.")
 	verbose := cmdTest.Flags().BoolP("verbose", "v", false, "Log all tests as they are run. Also print all text from Log and Logf calls even if the test succeeds.")
+	output := cmdTest.Flags().StringP("output", "o", "", "write to output file instead of executing the test directly")
```

Comment:
Hm. I just realised that in the Go tool, it's a combination of `-c` and `-o`. `-c` to "Compile the test binary to pkg.test but do not run it" and `-o` to overwrite the default of where the file should be written to. I still think we should imitate the Go tool, though.